<launch>
	
	<!-- Baxter world launch  -->
	
	<!-- <arg name="model" default="$(find baxter_description)/urdf/baxter.urdf.xacro"/> -->

	<node name="scenario_data_node" pkg="environment" type="scenario_data_node" />
	<node name="physical_agent_node" pkg="agent" type="physical_agent_node" />

	<node name="publish_environment_node" pkg="environment" type="publish_environment_node" />
	<node name="load_environment_node" pkg="environment" type="load_environment_node" />
	
	<node name="agent_brain" pkg="agent" type="agent_brain" />
	
</launch>
